webpack.config.dev.js

```diff
@@ -45,7 +48,7 @@ const config = {
     new ExtractTextPlugin('styles.css'),
     new webpack.DefinePlugin({
       'runtime.FHIR_URL': JSON.stringify(process.env.FHIR_URL || 'https://api.hspconsortium.org/cdshooksdstu2/open')
```

Comment:
Who sets this FHIR_URL on the environment?

Reply:
We can set this in the express node app that runs this Sandbox in Heroku, if needed. This was existing configuration from the previous sandbox.